deepmd/kernels/cute/neo/k1_kernels/cute_neo_so2_gate_combined_fwd.py

```
@@ -0,0 +1,779 @@
+
+    @cute.jit
+    def __call__(
+        self,
+        mX: cute.Tensor,
+        mResidual: cute.Tensor,
+        mW0: cute.Tensor,
+        mWP: cute.Tensor,
+        mGate: cute.Tensor,
+        mY: cute.Tensor,
+        mOut: cute.Tensor,
+        stream: cuda.CUstream = cuda.CUstream(cuda.CUstream_flags.CU_STREAM_DEFAULT),
```

Ruff (0.16.1)

[warning] Do not perform function call `cuda.CUstream` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)